src/components/ContactsAdd.js

```diff
@@ -167,9 +167,12 @@ function ContactsAdd(props) {
       />
 
       <div className="actions-section">
-        <button className="button blue" type="submit">
-          {location.state.id ? 'Edit' : 'Create'}
-        </button>
+        {location.state===null && <button className="button blue" type="submit">
+          Create
+        </button>}
```

Comment:
This is really nice Will! Conditionally rending the text inside the button depending on what it is being used for is really cool!